includes/class-admin.php

```diff
@@ -811,7 +811,8 @@ public static function row_actions( $actions, $post ) {
 			! \post_type_supports( \get_post_type( $post ), 'activitypub' ) ||
 			! in_array( $post->post_status, array( 'pending', 'draft', 'future', 'publish' ), true ) ||
 			! \current_user_can( 'edit_post', $post->ID ) ||
-			ACTIVITYPUB_CONTENT_VISIBILITY_LOCAL === get_content_visibility( $post )
+			ACTIVITYPUB_CONTENT_VISIBILITY_LOCAL === get_content_visibility( $post ) ||
+			site_supports_blocks()
```

Comment:
@pfefferle Are you interested in having more fine control here? Sometimes plugins offering custom post types to also not use the block editor yet for their post types, so whether the button is shown could be depending on that as well.

Reply:
@Menrath sure, can you create a PR for that?